serverlessworkflow/antora.yml

```diff
@@ -36,8 +37,9 @@ asciidoc:
     # only used in downstream
     kogito_devservices_imagename: registry.redhat.io/openshift-serverless-1-tech-preview/logic-data-index-ephemeral-rhel8
     kogito_examples_repository_url: https://github.com/kiegroup/kogito-examples
-    kogito_sw_examples_url: https://github.com/kiegroup/kogito-examples/tree/main/serverless-workflow-examples
+    kogito_sw_examples_url: https://github.com/kiegroup/kogito-examples/tree/{kogito_upstream_branch}/serverless-workflow-examples
```

Comment:
@ricardozanini do you know how to reference an attribute inside another attribute? from the docs it seems it should be in this way, but it didnt work 🤔 
in the and I can hardcode the version in the URL if there is no solution...

Reply:
Unfortunately, Antora doesn't support it. The CI will change that once we do a release. cc @radtriste 

Just hardcode the version for now.

Reply:
yep, the kgoito prepare release pipeline will change the kogito exampels URL

Reply:
> I think we can leave this as main, it will be changed to 1.27.x when forking, right?

yeah, good point @MarianMacik , the timeout example and job-service will be available just in community so it wont be cherry picked to 1.27.x